contrib/packs/actions/pack_mgmt/info.py

```diff
@@ -0,0 +1,16 @@
+from st2actions.runners.pythonrunner import Action
+import json
+import os
+
+GITINFO_FILE = '.gitinfo'
+
+
+class PackInfo(Action):
+    def run(self, pack, pack_dir="/opt/stackstorm/packs"):
+        gitinfo = os.path.join(pack_dir, pack, GITINFO_FILE)
+        try:
+            with open(gitinfo) as data_file:
+                details = json.load(data_file)
+                return details
+        except:
+            print "Unable to load git info for {}".format(pack)
```

Comment:
Hm, why don't we let exception bubble up or throw a custom one with a more friendly error message (Unable to load pack info for foo: %s)?

Not we don't let it bubble up and we don't exit with non-zero which means action will be considered as succeeded even on failure. Is this intentional?